regparser/web/jobs/utils.py

```diff
@@ -0,0 +1,91 @@
+from regparser.tasks import run_eregs_command
+import django_rq
+import settings
+
+
+def queue_eregs_job(args, timeout=60*30, result_ttl=-1):
+    """
+    Adds an eRegs command to the Redis queue to be run asynchronously.
+
+    Side effects
+        Adds a job to the Redis queue; eventually runs a job that alters the
+        data stored by the parser.
+
+    :arg list args: The commands for the eRegs command.
+    :arg int timeout: (Optional) How long (in seconds) the job should run
+        before being considered "lost".
+    :arg int result_ttl: (Optional) How long (in seconds) the key for the job
+        result should be stored.
+
+    :rtype: uuid4
+    :returns: The UUID for the job.
+    """
+    return django_rq.enqueue(run_eregs_command, args, timeout=60*30,
+                             result_ttl=-1)
+
+
+def delete_eregs_job(job_id):
+    """
+    Removes a job from the Redis queue.
+
+    Side effects
+        Removes a job from the Redis queue.
+
+    :arg uuid4 job_id: The UUID for the job.
+
+    :rtype: None
+    :returns: None
+    """
+    queue = django_rq.get_queue()
+    redis_job = queue.fetch_job(str(job_id))
+    if redis_job:
+        redis_job.delete()
+
+
+def add_redis_data_to_job_data(job_data):
```

Comment:
This feels a little fishy -- it's trying to synchronize with an asynchronous thing. I bet `rq` has some sort of life-cycle hooks that we could use instead.